common/index.ts

```diff
@@ -0,0 +1,40 @@
+import AdmZip from 'adm-zip';
+import fs from 'fs';
+import fetch from 'isomorphic-fetch';
+
+export enum ProjectSource {
+	FILE = 'file',
+	URI = 'uri',
+	CLOUD_ID = 'cloudId',
+	PROJECT_JSON = 'project.json',
+	MIT_PROJECT_URI = 'https://projects.scratch.mit.edu'
+}
+
+/*
+  The AssetFetcher is responsible for handing different
+  project sources and the asset files within or referred
+  to in a project.
+*/
+export const AssetFetcher = {
+	parseSb3: function (sb3File: string) {
```

Comment:
Hmm. In the future we probably want to introspect the content and not the filename.